sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/encoders/RowEncoder.scala

```diff
@@ -291,12 +296,22 @@ object RowEncoder {
           MapObjects(deserializerFor(_), input, et),
           "array",
           ObjectType(classOf[Array[_]]), returnNullable = false)
-      StaticInvoke(
-        scala.collection.mutable.WrappedArray.getClass,
-        ObjectType(classOf[Seq[_]]),
-        "make",
-        arrayData :: Nil,
-        returnNullable = false)
+      if (Utils.isScala212) {
+        StaticInvoke(
+          scala.collection.mutable.WrappedArray.getClass,
+          ObjectType(classOf[Seq[_]]),
+          "make",
+          arrayData :: Nil,
+          returnNullable = false)
+      } else {
+        // Use ArraySeq.unsafeWrapArray instead of WrappedArray.make in Scala 2.13
+        StaticInvoke(
```

Comment:
I am not sure can we use `scala.collection.immutable.ArraySeq.unsafeWrapArray` instead of `scala.collection.mutable.ArraySeq.make`, but `RowEncoderSuite` suite was fixed for this.